lesson01/hotel/src/com/andersenhotels/view/common/exception/WrongMenuChoiceException.java

```diff
@@ -0,0 +1,7 @@
+package com.andersenhotels.view.common.exception;
```

Comment:
move to the other exceptions

Reply:
i've moved all exceptions to presenter package
hope, that this is good desicion